### DISCUSSION TITLE
2 b2500 Akkus an HMS 1600 4T

### DISCUSSION THREAD

**Discussion by benyo84** [hoylabs/OpenDTU-OnBattery#1130] [Q&A]
Guten Morgen zusammen,

Ich habe folgendes Setting:
2x Marstek b2500 Speicher mit je 2,24kwh an den Wechselrichter angeschlossen HMS 1600 4T
also je 2 Solarmodule pro Speicher.
Tasmota Lesekopf und Open DTU on Battery zur Nulleinspeisung
In der Open DTU on Battery gibt es die Option bei Dynamic Power Limiter: Wechselrichter wird von Solarmodulen gespeist.
Ich denke diesen muss ich jetzt deaktivieren, da ich ja nun 2 Speicher dazwischen habe.
Welche Einstellungen muss ich bei Batterieschwellwerten eingeben?
oder kann ich es einfach auf Solarmodule stehen lassen? wäre aber doch bestimmt nicht gut für den Speicher, da er ihn immer komplett leer lutschen würde

Batterienutzung Start-Schwellwert

Batterienutzung Stop-Schwellwert

![Screenshot 2024-07-24 094601](https://github.com/user-attachments/assets/246df124-917a-46cb-adc6-d71d52cb3be1)

**Comment by Manos1966** [comment]
Hallo @benyo84 und willkommen!

Ganz schnell, ein paar Bemerkungen:
- Batterienutzung Start-Schwellwert 35V und gleichzeitig Batterienutzung Start-Schwellwert 35V ist nicht gut, du muss die Spannungswerte deiner Batterie(n) im Handbuch pruefen und passende Volt Werte fuer den Start und das Ende der Batterie Nutzung eingeben)
- Zwei gleiche Batterien, der Dynamic Power Limiter wird nur die Werte EINER Batterie verfolgen koennen (nur zur Info, kein Problem vorausgesetzt beide Batterien werden glaichmaessig geladen/entladen werden) 
- Minimales Leistungslimit zu klein, auf 55W mindestens einstellen

Es gibt viele Erklaerungen der Einstellungen im Wiki:
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter

https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter-%28Nulleinspeisung%29-%E2%80%93-MIT-BATTERIE

**Comment by benyo84** [comment]
Hallo Manos,

die Batterien sollten gleichmäßig laden, da alle 4 Paneele gleich ausgerichtet sind.

Technische Daten habe ich nur folgende gefunden
Technische Daten
Batterietyp: Lithium-Eisenphosphat (LiFePO4)
Zyklusleistung: 6000 Zyklen
Herstellergarantie: 3 Jahre
Batteriekapazität: 2240 Wh
Maximale Eingangsleistung: 1100 W
Maximale Ausgangsleistung: 800 W
Eingangsspannungsbereich: 18–55 V
Ausgangsspannungsbereich: 20–50 V
Maximale Anzahl parallel geschalteter Batterien: 5
Maximal erweiterbare Kapazität: 6720 Wh

**Comment by 3joern** [comment]
Lass es auf Solar im DPL stehen, das Leistungslimit schießt den WR immer ab bei diesen Akkus. Stelle in der Marstek App DOD auf 90 oder 95, dann bleiben 10 oder 5% in den Akkus. 
Mich wundert dass Du 2 komplette marstek geholt hast und nicht einen normalen mit Erweiterungsspeicher (ist so 200€ günstiger bei Hornbach), Leistung wäre gleich und die Regelung wäre nur über eine App. Ausser Du willst max 1600W Einspeisen, was evt nicht klug über ne Steckdose ist.

**Reply by spcqike** [reply]
> was evt nicht klug über ne Steckdose ist.

warum? 

Solange es angemeldet und entsprechend abgesichert ist, bzw. sicher gestellt ist dass keine weiteren Verbraucher dran sind. 

Mein HMS-2000 lief auch schon 7h auf 2kW. 

Über 2kW würd ich dennoch nicht gehen, da Schuko da nicht für Dauerlast gedacht is. Aber bis 2kW ist es ok.

**Comment by benyo84** [comment]
Hey ich habe die beiden speicher zu je 599€ bekommen und wie von euch
vermutet geht es mir um den maximalen output. Die Anlage ist von einem
elektriker direkt angeschlossen wurden und nicht über eine Steckdose, Von
daher kein problem. Das mit dem Dod werde ich ändern, aktuell lass ich ihn
immer komplett leer laufen. Aber an sich funktioniert es echt super so.

spcqike ***@***.***> schrieb am Fr., 2. Aug. 2024, 13:26:

**Comment by talutus** [comment]
Hallo benyo84,
wie ist denn Deine Erfahrung mit 2 B2500 Speichern an einem HMS-1600-4T ?
Konntest Du irgendwelche Probleme mit den Werten bzw. Spannungen beobachten?
Beim HM-1500 gab es ja wohl Probleme durch die zusammengeschalteten Massen.

**Comment by benyo84** [comment]
Moin,
Läuft so seit july tadellos, konnte keinerlei Probleme feststellen

talutus ***@***.***> schrieb am Di., 5. Nov. 2024, 15:25:

**Reply by Github-Nutzer** [reply]
@benyo84 - wie hast du denn das verschaltet? Speicher 1 auf HMS Eingang 1 und 2, oder 1 und 3, ...? 
Danke!

**Comment by talutus** [comment]
Sehr gut. Danke.

**Comment by stamas99** [comment]
Hallo, wenn ich zwei Batterien habe, aber mit ungleicher Ladung, nehme ich an, dass ich zwei HMS800 für Nulleinspeisung (mit Shelly und OpenDTU) benötigen würde?
Danke

**Comment by Gururom** [comment]
Normal sollten je zwei Module in einen marstek am HMS 1600 4t vollkommen problemlos laufen.
In der opendtuoB sind zwar nicht mehr die einuelnrn Module zu sehen, aber ansonsten regelt sich das alles wie selbst.
Im DPL einfach alles auf Solarmodule stehen lassen, der Akku gaukelt das dem wechselrichter dann vor.

**Comment by fm7775** [comment]
welche Einstellung muss vornehmen, um ein oder sogar zwei B2500 in openDTU oB zu bekommen. 

meine DTU Fusion habe ich in mehreren Fehlversuchen nun erfolgreich am laufen. hier liegen noch drei leere ESP-Wroom-32 rum,

**Comment by Gururom** [comment]
wenn du die aktuelle Version drauf hast, stell ein "durch Pufferbatterie gespeist" und dann sollte alles laufen!
